tasks/ghost-os.yml

```diff
@@ -4,3 +4,9 @@
   template: src=etc/init/ghost.conf.j2 dest=/etc/init/ghost.conf
   notify: restart ghost
   tags: [ghost]
```

Comment:
The thing is, I think you can run upstart on 15.04 and 15.10, just the default changed. Figured it would not hurt to also copy the file for those guys?